var/spack/repos/builtin/packages/py-pygit2/package.py

```diff
@@ -23,17 +25,20 @@ class PyPygit2(PythonPackage):
     depends_on("py-setuptools", type="build")
     # Version must match with libgit2
     # See: https://www.pygit2.org/install.html
-    depends_on("libgit2@1.1.0:1.1", when="@1.4:")
-    depends_on("libgit2@1.0.0:1.0", when="@1.2:1.3")
-    depends_on("libgit2@0:1.0", when="@1.1.0:1.1")
-    depends_on("libgit2@0.28.0:0.28", when="@0.28:1.0")
-    depends_on("libgit2@0.24:0.27", when="@0.24:0.27")
-    depends_on("python@3.6:", when="@1.4.0:")
-    depends_on("python@3.6:3.8", when="@1.2:1.3")
-    depends_on("python@3.5:3.8", when="@1.0:1.1")
-    depends_on("python@2.7:3.7", when="@0.28:0")
+    depends_on("libgit2@1.6", when="@1.12")
+    depends_on("libgit2@1.5", when="@1.11")
+    depends_on("libgit2@1.1", when="@1.4")
+    depends_on("libgit2@1.0", when="@1.2:1.3")
+    depends_on("libgit2@:1.0", when="@1.1")
+    depends_on("libgit2@:0.28", when="@:1.0")
+    depends_on("libgit2@:0.27", when="@:0.27")
+    depends_on("python@3.8:", when="@1.11.1")
+    depends_on("python@3.7:", when="@1.4.0:1.6.0")
+    depends_on("python@:3.8", when="@1.0:1.3")
+    depends_on("python@3.7", when="@0.28:0")
     depends_on("py-six", type=("build", "run"), when="@:0.28.2")
-    depends_on("py-cffi@1.4.0:", type=("build", "run"))
+    depends_on("py-cffi@1.9.1:", when="@1.11.1:", type=("build", "run"))
+    depends_on("py-cffi@1.4.0:", when="@:1.6.0", type=("build", "run"))
     depends_on("py-cached-property", when="@1.1.0:1.5", type=("build", "run"))
     depends_on("py-cached-property", when="@1.6.0: ^python@:3.7", type=("build", "run"))
```

Comment:
Seems like this dep is no longer required

Reply:
change made

Reply:
Did you forget to push the commit? The package still lists cached-property as required

Reply:
I am not sure what happened; I just did another push.  There should not have been any more commits, but there seemed to be.  The screen I am on right now is too small to see anything.  When I get to a larger screen, I will check what those commits were. I hope I didn't create a mess.

Reply:
Doesn't appear to be any changes to py-cached-property